packages/accessible-react-app/src/js/layout/Layout.js

```diff
@@ -12,7 +12,7 @@ const Layout = props => {
   return (
     <>
       {/* FE Talking Point - React.Fragment for cleaner HTML */}
-      {/* FE Talking Point - WCAG SC 2.4.2: Page Titled */}
+      {/* SR Talking Point - WCAG SC 2.4.2: Page Titled */}
```

Comment:
Some URLs on these might be nice for further reading: https://www.w3.org/WAI/WCAG21/quickref/#page-titled